scss/utilities/_sizing.scss

```diff
@@ -1,8 +1,15 @@
 // Width and height
 
-@each $prop, $abbrev in (width: w, height: h) {
-  @each $size, $length in $sizes {
-    .#{$abbrev}-#{$size} { #{$prop}: $length !important; }
+@each $breakpoint in map-keys($grid-breakpoints) {
+  @include media-breakpoint-up($breakpoint) {
+    $infix: breakpoint-infix($breakpoint, $grid-breakpoints);
+
+    @each $prop, $abbrev in (width: w, height: h) {
+      @each $size, $length in $sizes {
+        .#{$abbrev}-#{$size} { #{$prop}: $length !important; }
```

Comment:
@andresgalante it seems this line is not needed
when `$breakpoint` is `xs`, `$infix` is an empty string

Reply:
Adding sizing for all breakpoints I use.
```scss
// Create sizing classes for all screen sizes
@each $breakpoint in map-keys($grid-breakpoints) {
  $infix: breakpoint-infix($breakpoint, $grid-breakpoints);

  @if $infix != '' {
    @include media-breakpoint-up($breakpoint) {
      @each $prop, $abbrev in (width: w, height: h) {
        @each $size, $length in $sizes {
          .#{$abbrev}#{$infix}-#{$size} { #{$prop}: $length !important; }
        }
      }
    }
  }
}
```

Reply:
@iamandrewluca: please make a PR; commenting in older PRs/issues will probably be ignored.

Reply:
@XhmikosR I just wanted to mention this to @andresgalante . As I see this is not being merged. Should I create a PR in any case?

Reply:
@iamandrewluca you are right, thanks a lot for reviewing it. 😄